packages/site/src/content/Avatar/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Avatar } from "@jobber/components";
+import Content from "@atlantis/docs/components/Avatar/Avatar.stories.mdx";
+import Props from "./Avatar.props.json";
+import { ContentExport } from "../../types/content";
+
+export default {
+  content: Content,
```

Comment:
I think the file needs to be a `.tsx` file and this should be rewritten as
```suggestion
  content: () => <Content />,
```
to match the other files. It looks like the import is called `ComponentContent` so that may want to be updated as well

Reply:
Also minor but I notice I get a diff in `Disclosure.props.json` so I think we may want to commit that diff either in this PR or another